packages/arb-token-bridge-ui/src/util/deposits/fetchDeposits.ts

```diff
@@ -126,11 +125,5 @@ export const fetchDeposits = async ({
     }
   )
 
-  const finalTransactions: Transaction[] = await Promise.all(
-    ethDepositsFromSubgraph.map(depositTx =>
-      updateAdditionalDepositData(depositTx, l1Provider, l2Provider)
-    )
-  )
-
-  return finalTransactions
+  return ethDepositsFromSubgraph
```

Comment:
I'm guessing this is just a misnomer and this actually includes token deposits, too?